src/app/components/login/loginModal.html

```diff
@@ -16,10 +16,10 @@ <h2>Log in</h2>
           <h2>People looking for a better city</h2>
           <p>You're part of them? Feel free to join us!</p>
         </div>
-        <div layout="row" layout-sm="column">
```

Comment:
Making sure the `username` and `password` are not side by side, but above and below one another.